src/os/bluestore/BitMapAllocator.cc

```diff
@@ -221,8 +220,8 @@ void BitMapAllocator::init_rm_free(uint64_t offset, uint64_t length)
            << " length 0x" << length << std::dec
            << dendl;
 
-  assert(!(offset % m_block_size));
```

Comment:
Offset and length needs to be multiple of block size, not necessarily power of two, please revert all changes related to that.